src/IntentBuilder.ts

```diff
@@ -1,9 +1,26 @@
 import { BytesLike, ethers } from 'ethers';
-import { BUNDLER_URL, CHAIN_ID, ENTRY_POINT, FACTORY, NODE_URL } from './Constants';
+import { BUNDLER_URL_MAIN, BUNDLER_URL_LIVE, BUNDLER_URL_DEV, CHAIN_ID, ENTRY_POINT, FACTORY, NODE_URL } from './Constants';
 import { Client, Presets, UserOperationBuilder } from 'userop';
 import { Intent } from 'blndgs-model/dist/asset_pb';
 
+const env = process.env.env;
+
+let BUNDLER_URL:string;
+
+if(env === 'LIVE') {
+  BUNDLER_URL = BUNDLER_URL_LIVE
+}
+if(env === 'MAIN') {
+  BUNDLER_URL = BUNDLER_URL_MAIN
+}
+if(env === 'DEV') {
+  BUNDLER_URL = BUNDLER_URL_DEV
+}
```

Comment:
The `main` branch points to the development environment
`test` branch -> test
[/balloondogs/Environments-Map](https://www.notion.so/native//balloondogs/Environments-Map-29e0e865b2d64fd0b3c1b6279835a645?pvs=4)